packages/oui-tile/src/tile.html

```diff
@@ -0,0 +1,7 @@
+<div class="oui-tile">
+    <h4 class="oui-header_4 oui-tile__title" ng-bind="::$ctrl.heading"></h4>
+    <p class="oui-tile__legend" ng-if="!!$ctrl.description" ng-bind="::$ctrl.description"></p>
+    <div class="oui-tile__body">
+        <div ng-transclude></div>
```

Comment:
(optional) Why not setting the transclude directly on `oui-tile__body` ?

Reply:
Yup u right!